### PR TITLE
Add blog post about J5 v1.0

### PR DESCRIPTION
**Pull request by lyzadanger**
Hello! This PR adds a (draft) blog post about the release of Johnny-Five 1.0. Please do not merge as-is! We need to (at the least) determine whether we're going to tag another pre-release before 1.0, and @nebrius should get a change to do editorial revision.

### CODE REVIEW

File: src/news/johnny-five-1-point-0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+author: Bryan Hughes and Lyza Danger Garder
```
Comment (lyzadanger) [inline src/news/johnny-five-1-point-0.md]: Is dual authorship going to break anything in build?

File: src/news/johnny-five-1-point-0.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+title: 'Johnny-Five v1.0'
+slug: 'v1_0'
+category:
+  - Announcement
+  - Platform
+---
+
+Hi from [JSConf US](https://2018.jsconf.us/) in sunny San Diego, Calif., where a representative group of Johnny-Five contributors [met and planned](https://github.com/rwaldron/johnny-five/issues/1463) what's next for Johnny-Five. And—ta-da!—we are thrilled to announce the release of Johnny-Five v1.0!
+
+Don't panic. This is the Johnny-Five you already know and love. There are no breaking changes from the last released, tagged version (???0.15.1). In fact, there are zero changes at all.
```
Comment (lyzadanger) [inline src/news/johnny-five-1-point-0.md]: Alert! We need to determine whether we're going to tag `0.15.1` before `1.0`

File: src/news/johnny-five-1-point-0.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+But! Now we've got our [semver](https://semver.org/) wheels in motion. We're in a good position to start kicking out some of the features we've been excited about (but have been scared to ship because some are breaking).
+
+We're committing to the support of both [LTS (Long-Term Support) branches of Node.js](https://github.com/nodejs/Release), which, at time of versioning are: 8.x (active LTS) and 6.x (maintenance LTS). Already succesfully running J5 on an older version of Node.js? Not to worry: it won't stop working. We're just being more thoughtful and explicit about our Node.js version support going forward.
```
Comment (lyzadanger) [inline src/news/johnny-five-1-point-0.md]: Is this true?!

**Comment by lyzadanger** [comment]
Aha! Apparently I can't assign reviewers. @nebrius and @rwaldron HELLO ALERT! 😄

File: src/news/johnny-five-1-point-0.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+date: '2018-08-23 00:00:00'
+status: published
+title: 'Johnny-Five v1.0'
+slug: 'v1_0'
+category:
+  - Announcement
+  - Platform
+---
+
+Hi from [JSConf US](https://2018.jsconf.us/) in sunny San Diego, Calif., where a representative group of Johnny-Five contributors [met and planned](https://github.com/rwaldron/johnny-five/issues/1463) what's next for Johnny-Five. And—ta-da!—we are thrilled to announce the release of Johnny-Five v1.0!
```
Comment (lyzadanger) [inline src/news/johnny-five-1-point-0.md]: Should we link `Johnny-Five v1.0` to the (to-be-released) link on `npm`?

Comment (nebrius) [reply src/news/johnny-five-1-point-0.md]: I kinda think we should list it as `1.0.0` instead of `1.0` to help drive home that this is about semver. What do you think?

File: src/news/johnny-five-1-point-0.md (hunk 3 above)
Comment (nebrius) [reply src/news/johnny-five-1-point-0.md]: I would reword this to be more like "it won't stop working overnight, however we may make modifications in the future that cause Johnny-Five to break on unsupported versions of Node.js" or something to that effect. Once we start using more ES6 features, we'll slowly start breaking on older versions and I don't want to give the impression otherwise.

Comment (nebrius) [reply src/news/johnny-five-1-point-0.md]: The LTS wording looks spot on!

File: src/news/johnny-five-1-point-0.md (hunk 4 above)
Comment (dtex) [reply src/news/johnny-five-1-point-0.md]: Agreed on ```1.0.0```

File: src/news/johnny-five-1-point-0.md (hunk 3 above)
Comment (dtex) [reply src/news/johnny-five-1-point-0.md]: Agree with @nebrius regarding wording and explicitly mentioning inevitable ES6 changes that will come soon.

File: src/news/johnny-five-1-point-0.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+category:
+  - Announcement
+  - Platform
+---
+
+Hi from [JSConf US](https://2018.jsconf.us/) in sunny San Diego, Calif., where a representative group of Johnny-Five contributors [met and planned](https://github.com/rwaldron/johnny-five/issues/1463) what's next for Johnny-Five. And—ta-da!—we are thrilled to announce the release of [Johnny-Five v1.0.0](https://www.npmjs.com/package/johnny-five/v/1.0.0)!
+
+Don't panic. This is the Johnny-Five you already know and love. There are no breaking changes from the most recent released, tagged version ([v0.15.1](https://github.com/rwaldron/johnny-five/releases/tag/v0.15.1)). In fact, there are [zero changes at all](https://github.com/rwaldron/johnny-five/releases/tag/v1.0.0).
+
+We're committing to the support of both [LTS (Long-Term Support) branches of Node.js](https://github.com/nodejs/Release), which, at time of versioning are: 8.x (active LTS) and 6.x (maintenance LTS). Already succesfully running J5 on an older version of Node.js? That's OK; it's not going to stop working overnight. We are, however, marching boldly forward: be forewarned that future releases may break on unsupported versions of Node.js.
```
Comment (nebrius) [inline src/news/johnny-five-1-point-0.md]: I love this new wording, it’s so good!
